fixbackend/app.py

```diff
@@ -142,7 +145,21 @@ async def setup_teardown_application(app: FastAPI) -> AsyncIterator[None]:
         )
         deps.add(
             SN.customerio_consumer,
-            CustomerIoEventConsumer(http_client, cfg, readwrite_redis, domain_events_stream_name),
+            CustomerIoEventConsumer(http_client, cfg, readwrite_redis, DomainEventsStreamName),
+        )
+        cloud_accounts_redis_publisher = RedisPubSubPublisher(
+            redis=readwrite_redis, channel="cloud_accounts", publisher_name="cloud_account_service"
+        )
+        deps.add(
+            SN.cloud_account_service,
+            CloudAccountServiceImpl(
+                workspace_repo,
+                CloudAccountRepositoryImpl(session_maker),
+                cloud_accounts_redis_publisher,
+                domain_event_publisher,
+                JsonStoreImpl(session_maker),
```

Comment:
I wonder why there is *the* json store.
Don't we want to use different underlying tables based on use case?